C7/Game.cs

```diff
@@ -482,6 +482,25 @@ public override void _UnhandledInput(InputEvent @event)
 					UnitInteractions.moveUnit(CurrentlySelectedUnit.guid, dir);
 				}
 			}
+			else if ((eventKey.Scancode >= (int)Godot.KeyList.Home) && (eventKey.Scancode <= (int)Godot.KeyList.Pagedown))
+			{ // Move units with the arrow and fn keys
+				if (CurrentlySelectedUnit != MapUnit.NONE)
+				{
+					TileDirection dir;
+					switch (eventKey.Scancode) {
```

Comment:
Can we get the direction from a map instead of a switch on the eventKey? It could be nice to have the key mappings less tightly coupled with the unit movement logic in case we want to change them or read them from a file in the future

Reply:
Key bindings and configuration could easily be a task of its own, but while we're on the topic it would be good to at least decouple the specific keys to somewhere that we can make configurable later.

Reply:
Created https://github.com/C7-Game/Prototype/issues/168 to track this

Reply:
I'll see about implementing that in this PR

Reply:
Since it's a relatively large feature it might be nice to merge it in a separate PR so it's easier to find feature changes in version control. Either way, feel free to assign yourself to the ticket if you decide to implement it